exporters/tests/test_failure_exporter.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+#
+# Copyright Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+#
+
+import pytest
+from _pytest.monkeypatch import MonkeyPatch
```

Comment:
`MonkeyPatch` is exposed within `pytest`, no need to look for it in a private module.

Reply:
@KevinMGranger normally yes by fixture, but for some reason pytest was failing that the module was not found.